src/core/show.ml

```diff
@@ -1,5 +1,5 @@
 
 module type S = sig
```

Comment:
Done, I am thinking to add a new module as well to complete the `fromString/toString` cycle, do you think we should call it `Parsable`? Or any other name you prefer?